nexus-repository-r-it/src/test/java/org/sonatype/nexus/plugins/r/internal/RITSupport.java

```diff
@@ -82,6 +86,8 @@
 
   public static final String TARGZ_EXT = ".tar.gz";
 
+  public static final String TRGZ_EXT = ".trgz";
```

Comment:
make wrong extension more obvious

Reply:
Done in 1526201bff35a45a7383786942c14c02aa3015c7